src/containers/blocks.jsx

```diff
@@ -155,7 +165,6 @@ class Blocks extends React.Component {
                 this.setLocale();
             } else {
                 this.props.vm.refreshWorkspace();
-                this.updateToolbox();
```

Comment:
Do you know what this case was covering before? I'm confused about the `if (this.props.isVisible === prevProps.isVisible)` check we're in.